lib/encoding-status-view.js

```diff
@@ -64,6 +68,11 @@ export default class EncodingStatusView {
         this.encodingLink.textContent = encodingLabel.status
         this.encodingLink.dataset.encoding = editorEncoding
         this.element.style.display = ''
+
+        if(this.tooltip) {
+          this.tooltip.dispose()
+        }
+        this.tooltip = atom.tooltips.add(this.encodingLink, {title: 'File displayed using ' + encodingLabel.list + ' encoding'})
```

Comment:
Very good point.  Changed to `This file uses UTF-8 encoding`.

I wasn't sure I wanted to be the first to use interpolation in this package... but it works fine.  :)